tests/model/test_agent.py

```diff
@@ -1,32 +1,43 @@
-from tests.base import TestCase, assets, main # pylint: disable=unused-import
+# -*- coding: utf-8 -*-
 
-from ocrd_models import OcrdAgent
+from ocrd_models import (
+    OcrdAgent
+)
+from tests.base import (
+    main
+)
 
-# pylint: disable=no-member
-class TestOcrdAgent(TestCase):
 
-    def test_basic1(self):
-        ag = OcrdAgent(role='FOO')
-        self.assertEqual(ag.role, 'FOO')
-        self.assertEqual(ag.name, None)
-        self.assertEqual(str(ag), '<OcrdAgent [type=---, othertype=---, role=FOO, otherrole=---, name=---]/>')
+def test_init_role_no_name():
+    ag = OcrdAgent(role='FOO')
+    assert ag.role == 'FOO'
+    assert ag.name is None
 
-    def test_basic2(self):
-        ag = OcrdAgent(otherrole='BAR', othertype='x')
-        self.assertEqual(ag.role, 'OTHER')
-        self.assertEqual(ag.otherrole, 'BAR')
-        self.assertEqual(ag.othertype, 'x')
 
-    def test_basic3(self):
-        ag = OcrdAgent(name='foobar')
-        self.assertEqual(ag.name, 'foobar')
-        ag.name = 'barfoo'
-        self.assertEqual(ag.name, 'barfoo')
+def test_init_role_as_string():
+    ag = OcrdAgent(role='FOO')
+    expected = '<OcrdAgent [type=---, othertype=---, role=FOO, otherrole=---, name=---]/>'
+    assert expected == str(ag)
```

Comment:
Won't that take away the benefit of seeing the expected value along with the actual value whenever the assertion fails? (`pytest.assertEqual` shows both IIRC, but plain assertions only show the line of code, which does not even contain the literal value anymore here.)

Reply:
pytest does some black magic behind the scenes to turn "plain" assertions:

```python
    def test_foo():
        expected = [1,2,3]
        actual = {"foo": "bar"}
>       assert expected == actual
E       AssertionError: assert [1, 2, 3] == {'foo': 'bar'}
E         +[1, 2, 3]
E         -{'foo': 'bar'}
E         Full diff:
E         - {'foo': 'bar'}
E         + [1, 2, 3]
```

Reply:
Ah, thanks!